lib/lita/handlers/google.rb

```diff
@@ -33,7 +36,22 @@ def search(response)
 
         if http_response.status == 200
           data = MultiJson.load(http_response.body)
-          result = data["responseData"]["results"].first
+
+          if config.excluded_domains
+            #reject all exlcuded domains
+            data["responseData"]["results"].each do |res|
+              url = res['unescapedUrl']
+              url = url.gsub("http://www.", "www.").gsub("http://", "www.")
+              if config.excluded_domains.include? url.scan(/^(?:www\.)?(.*?)\.(?:com|au\.uk|co\.in)/).flatten[0]
+                next
+              else
+                result = res
+                break
+              end
+            end
+          else
+            result = data["responseData"]["results"].first
+          end
```

Comment:
Use [Enumerable#find](http://ruby-doc.org/core-2.2.2/Enumerable.html#method-i-find) instead of this procedural style.

We should require the TLD(s) in addition to the domain name for each so that there's no need to do all this URL substiution. Use `URI.parse` to get a URI object from each URL and then just use `#include?` on the host part. Make sure you handle the (unlikely) case that `URI.parse` raises an exception due to a bad URI.